index.html

```diff
@@ -14,14 +14,190 @@
       href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;400&display=swap"
       rel="stylesheet"
     />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@700&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Raleway:wght@800&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Raleway:wght@400&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Raleway:wght@700&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@600&display=swap"
+      rel="stylesheet"
+    />
     <meta name="theme-color" content="#e30062" />
   </head>
-
   <body>
-    <header>
-      <nav>
-        <!-- in case of small screens, show a dropdown -->
-        <input type="checkbox" id="nav-toggle" aria-controls="navbar toggle" />
+    <div class="first-screen">
+      <header>
+        <img
+          id="logo"
+          src="../img\re-designed images\RDS-logo.png"
+          alt="logo"
+        />
+
+        <nav class="main-nav">
+          <a
+            class="nav-text welcome"
+            href="https://www.google.com"
+            target="_blank"
+            >Welcome</a
+          >
+          <a
+            class="nav-text events"
+            href="https://www.google.com"
+            target="_blank"
+            >Events</a
+          >
+          <a
+            class="nav-text member"
+            href="https://www.google.com"
+            target="_blank"
+            >Member</a
+          >
+          <a
+            class="nav-text crypto"
+            href="https://www.google.com"
+            target="_blank"
+            >Crypto</a
+          >
+          <a
+            class="nav-text status"
+            href="https://www.google.com"
```

Comment:
Href is not addressed properly here

Reply:
The re-designing of the whole website is yet to complete, and tasks have been assigned to other members who will be working on this code after this PR gets accepted. So hopefully, the links will be addressed properly once the whole re-designing process is complete.